### PR TITLE
SFMS follow up

### PR DESCRIPTION
**Pull request by Sybrand**
- Fix folder date bug (5pm runs would push the process date ahead because of UTC)
- Handling ; for comments in .ini file
# Test Links:
[Percentile Calculator](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2246.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

File: api/app/routers/sfms.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def is_actual(filename: str) -> bool:
         year=int(file_date_string[:4]),
         month=int(file_date_string[4:6]),
         day=int(file_date_string[6:8]))
-    now = get_pst_now()
+    now = get_pdt_now()
```
Comment (andrea-williams) [inline api/app/routers/sfms.py:55]: I'm confused - why the use of PDT here? Based on https://github.com/bcgov/wps/issues/1579, I thought using PDT was a big no-no?

Comment (Sybrand) [reply api/app/routers/sfms.py:55]: I know right! Timezone nightmares.
So. In this case. Here's the deal:
- If it's say, the 25th of August. Then it means we (citizens of bc) are using PDT.
- If we have a server, that makes a prediction, on the 25th of August, then we refer to that prediction as having run on the 25 of August.
- If we use a different timezone, then we may end up saying it was on a different date! So using PST, we may misattribute the date something happened on.


Such garbage. BC should just switch to PST as Glod intended.

Comment (andrea-williams) [reply api/app/routers/sfms.py:55]: I'm still not able to wrap my head around this, but I don't want to have to think about it anymore. I partially trust your logic on this, and if it turns out to be wrong, well then I'll just blame you.

**Review by andrea-williams — APPROVED** [review]

File: api/app/routers/sfms.py (hunk 1 above)
Comment (Sybrand) [reply api/app/routers/sfms.py]: Ok - switched to using "america/vancouver" - if you care to look again.

**Pull request merged**